src/ClusterManager/joblog_manager.py

```diff
@@ -40,42 +41,23 @@ def extract_job_log(jobId, logPath, userId):
     dataHandler = None
     try:
         dataHandler = DataHandler()
-
-        old_cursor_text = dataHandler.GetJobTextField(jobId, "jobLog")
-        if old_cursor_text is not None and old_cursor_text.startswith("$$cursor: "):
-            old_cursor = old_cursor_text[10:]
-        else:
-            old_cursor = None
-        (logs, new_cursor) = GetJobLog(jobId, cursor=old_cursor)
-
-        container_logs = {}
-        for log in logs:
-            try:
-                container_id = log["_source"]["docker"]["container_id"]
-                log_text = log["_source"]["log"]
-                if container_id in container_logs:
-                    container_logs[container_id] += log_text
-                else:
-                    container_logs[container_id] = log_text
-            except Exception:
-                logging.exception("Failed to parse elasticsearch log: {}".format(log))
+        lines = GetJobLog(jobId)
 
         jobLogDir = os.path.dirname(logPath)
         if not os.path.exists(jobLogDir):
             mkdirsAsUser(jobLogDir,userId)
 
-        for (container_id, log_text) in container_logs.items():
+        for containerId, containerLogLines in itertools.groupby(
+            lines,
+            key=lambda line: line["docker"]["container_id"]
+        ):
             try:
-                containerLogPath = os.path.join(jobLogDir, "log-conatainer-" + container_id + ".txt")
-                with open(containerLogPath, 'a') as f:
-                    f.write(log_text)
+                containerLogPath = os.path.join(jobLogDir, "log-conatainer-" + containerId + ".txt")
```

Comment:
@xudifsd if we decide to remove `/job`, do we still want write logs to files?

Reply:
No, we should remove this dump to NFS

Reply:
@Gerhut  Still didn't removed this?

Reply:
Not sure your plan, should I rebase dltsdev branch?

Reply:
Sorry, missed this. You can just remove this logic in both ES clusters and non-ES clusters. We are trying to remove dependency on NFS.